interface/src/app/analyze/analysis/analysis.js

```diff
@@ -29,9 +29,9 @@ angular.module('adage.analyze.analysis', [
   });
 }])
 
-.controller('AnalysisCtrl', ['$scope', '$log', '$location', '$q', 'Sample',
-'Activity', 'AnnotationType', 'SampleBin',
-function AnalysisCtrl($scope, $log, $location, $q, Sample, Activity,
+.controller('AnalysisCtrl', ['$scope', '$log', '$location', '$q', '$state',
+'Sample', 'Activity', 'AnnotationType', 'SampleBin',
+function AnalysisCtrl($scope, $log, $location, $q, $state, Sample, Activity,
```

Comment:
Hmmm - Are ``$location``, ``Sample``, and ``Activity`` being used in this controller? I don't think I see them below.